packages/ai-jsx/src/next/index.ts

```diff
@@ -0,0 +1,114 @@
+import 'server-only';
+import * as ReactModule from 'react';
+import * as LLMx from '../react/core.js';
+export * from '../react/core.js';
+import { markAsJsxBoundary } from '../react/jsx-boundary.js';
+
+function unwrapReact(partiallyRendered: LLMx.PartiallyRendered): ReactModule.ReactNode {
+  if (LLMx.isElement(partiallyRendered)) {
+    // This should be an AI.React element.
+    if (partiallyRendered.tag !== LLMx.React) {
+      throw new Error('unwrapReact only expects to see AI.React elements or strings.');
```

Comment:
Can we add one or two more sentences to help devs know how to fix this? And to make it clear whether it's a dev or internal problem? 

Imagine the person seeing this error is brand new to our framework and also their child is asking them for attention at this moment. :smile:

Reply:
It's an internal invariant being violated (i.e. an AI.jsx bug), either partial rendering something wrong or the boundary render stopping at element other than `AI.React`.

I'll prefix it to try to make it more clear that this is an AI.jsx problem.